Gemfile

```diff
@@ -2,7 +2,7 @@ require './prox_gem'
 
 prox_gem 'clock'
 prox_gem 'attribute'
-prox_gem 'null_attr'
+prox_gem 'subst_attr'
```

Comment:
It appears that the only prox-gem needed for this library is `runner`. The loader doesn't load any of these. This is probably something i did when copying the code from another library. @kizzle102 could you try removing the other prox-gems and see if the tests still run?